<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_MAKRO_Eilenstein</class>
 <widget class="QMainWindow" name="D_MAKRO_Eilenstein">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_Out">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_Out_Select">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_Out_Open">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_Out_Path">
         <property name="maximumSize">
          <size>
           <width>156</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Output Path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer_Out">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Ignored</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>153</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" rowspan="2">
        <spacer name="horizontalSpacer_Out">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_Run">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Run</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_Run_Stack">
         <property name="minimumSize">
          <size>
           <width>156</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>156</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Process X Images</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_Run">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Ignored</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>11</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_Run">
         <property name="minimumSize">
          <size>
           <width>156</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>156</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Process Stack</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <spacer name="horizontalSpacer_Run">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>73</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_Set">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLabel" name="label_Min">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_Max">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_Step">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="3">
        <spacer name="horizontalSpacer_Set">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>4</width>
           <height>62</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Size">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_Min">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_Max">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBox_Step">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Off">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Offset</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Off_Min">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-256.000000000000000</double>
         </property>
         <property name="maximum">
          <double>255.000000000000000</double>
         </property>
         <property name="value">
          <double>-16.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Off_Max">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-256.000000000000000</double>
         </property>
         <property name="maximum">
          <double>255.000000000000000</double>
         </property>
         <property name="value">
          <double>16.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_Off_Step">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>-256.000000000000000</double>
         </property>
         <property name="maximum">
          <double>255.000000000000000</double>
         </property>
         <property name="value">
          <double>4.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <spacer name="verticalSpacer_Set">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Ignored</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>321</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_In">
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_Img_Add">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Add Images</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+O</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_img_Clear">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Clear Images</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBox_Img"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QGraphicsView" name="graphicsView_Img"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>916</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
